test/query_test.rb

```diff
@@ -11,4 +11,15 @@ def test_basic
     assert_equal ["Apple", "Milk"], query.execute.map(&:name).sort
   end
 
+  def test_with_effective_min_score
+    store_names ["Milk", "Milk2"]
+
+    assert_equal ["Milk"], Product.search("Milk", min_score: 0.1).map(&:name)
```

Comment:
@ankane I added facets' result test, but I am not sure what you want to test in sql test. Won't it be a lot like query test?